.circleci/check_gotest.sh

```diff
@@ -27,17 +28,10 @@ for go_version in $GO_VERSIONS; do
     echo "GOROOT=$GOROOT"
     echo "PATH=$PATH"
     
-    go test -v -tags="${TEST_BUILD_TAGS}" ./...;
+    go test -tags="${TEST_BUILD_TAGS}" ${TEST_EXTRA_BUILD_FLAGS} ./...;
     status="$?"
     if [[ "${status}" != "0" ]]; then
-        echo "$version-tls12" >> "$FILEPATH_ERROR_FLAG";
-    fi
-
-    # test with supported tls1.3
-    GODEBUG="tls13=1" go test -v -tags="${TEST_BUILD_TAGS}" github.com/cossacklabs/acra/...;
```

Comment:
What this piece of script was supposed to do? TLS 1.3 is now the default one and we won't test with 1.2 anymore?

Reply:
https://go.dev/doc/go1.12#tls_1_3 - before we had to explicitly set `tls13=1` to turn on tls1.3 support. Now we test with 1.2 by default because all our tls.Config initialized with min version 1.2. But as I remember, golang uses the greatest supported version, so by default will be used 1.3. But we have some test cases where we test weak ciphersuits that we explicitly define on client side to check that our server side will deny such connections. And declaring ciphersuits are supported only with tls1.2.